### PR TITLE
Add robot.txt to ignore admin and c2c interface

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/views/entry.py [hunk 1]
```diff
@@ -59,7 +59,11 @@ def apijs(self):
         return {}
 
     def favicon(self):
-        set_common_headers(self.request, "api", NO_CACHE, content_type="image/vnd.microsoft.icon")
+        set_common_headers(self.request, "index", NO_CACHE, content_type="image/vnd.microsoft.icon")
+        return {}
+
+    def robot_txt(self):
+        set_common_headers(self.request, "index", NO_CACHE, content_type="text/txt")
```
Comment (fredj) [inline geoportal/c2cgeoportal_geoportal/views/entry.py]:
```suggestion
        set_common_headers(self.request, "index", NO_CACHE, content_type="text/plain")
```

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/views/entry.py]: :-)

File: geoportal/c2cgeoportal_geoportal/__init__.py [hunk 2]
```diff
@@ -449,15 +449,16 @@ def handle(event: InvalidateCacheEvent):  # pylint: disable=unused-variable
...
         config.add_route(name, path, request_method="GET")
         config.add_view(Entry, attr=attr, route_name=name, renderer=renderer)
 
-    add_api_route("favicon", "favicon", "/favicon.ico", "/etc/geomapfish/static/images/favicon.ico")
-    add_api_route("apijs", "apijs", "/api.js", "/etc/static-ngeo/api.js")
-    add_api_route("apijsmap", "apijsmap", "/api.js.map", "/etc/static-ngeo/api.js.map")
-    add_api_route("apicss", "apicss", "/api.css", "/etc/static-ngeo/api.css")
-    add_api_route("apihelp", "apihelp", "/apihelp/index.html", "/etc/geomapfish/static/apihelp/index.html")
+    add_static_route("favicon", "favicon", "/favicon.ico", "/etc/geomapfish/static/images/favicon.ico")
+    add_static_route("robot.txt", "robot_txt", "/robot.txt", "/etc/geomapfish/static/images/robot.txt")
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/__init__.py]: why not having robot.txt in `/etc/geomapfish/static` instead of `/etc/geomapfish/static/images/` ?

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/__init__.py]: Oups, thanks :-)

**Pull request merged**